tests/components/observers/test_household_survey_observers.py

```diff
@@ -0,0 +1,80 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from vivarium_census_prl_synth_pop.components.observers import HouseholdSurveyObserver
+
+# TODO: Broader test coverage
+
+
+@pytest.fixture
+def observer(mocker, tmp_path):
+    """Generate post-setup observer with mocked methods to patch as necessary"""
+    observer = HouseholdSurveyObserver("ACS")
+    builder = mocker.MagicMock()
+    builder.configuration.output_data.results_directory = tmp_path
+    observer.setup(builder)
+    return observer
+
+
+@pytest.fixture
+def mocked_pop_view(observer):
+    """Generate a state table view"""
+    cols = observer.input_columns
+    df = pd.DataFrame(np.random.randint(0, 100, size=(2, len(cols))), columns=cols)
+    df["alive"] = "alive"
+    df[["first_name", "middle_name", "last_name"]] = (
+        ["Alex", "J", "Honnold"],
+        ["Carolynn", "Marie", "Hill"],
+    )
+
+    return df
+
+
+def test_instantiate(observer):
+    assert str(observer) == "HouseholdSurveyObserver(ACS)"
+
+
+def test_responses_schema(observer):
+    """Is the initial self.responses (after setup) the expected schema
+    (pd.DataFrame with correct columns)?
+    """
+    pd.testing.assert_frame_equal(
```

Comment:
I didn't know about this function

Reply:
it's a good one and I use it all the time